3rdparty/CMakeLists.txt

```diff
@@ -18,7 +18,10 @@ set(CMAKE_CXX_FLAGS "${flags}")
 option(DOCTEST_NO_INSTALL "Skip the installation process" ON)
 add_subdirectory(doctest)
 
-option(JUSTRX_HAVE_BENCHMARK "Benchmark is provided" ON)
+set(DISABLE_TESTS ON)
+# If we'd enable tests above we would need to pick up the benchmarking
+# library here to avoid declaring the target twice.
+set(JUSTRX_HAVE_BENCHMARK ON)
```

Comment:
If you are wondering why this changed from a new option to setting the already existing variable, I made this change since it was slightly unusual to shadow an option from justrx (though it still worked on my CMake version); it is also different for how we treat any other 3rdparty library's flags. I fixed building of benchmarks in https://github.com/rsmmr/justrx/pull/15 so it should now be possible to build and run them there.